pkg/services/ngalert/api/api.go

```diff
@@ -76,6 +83,7 @@ func (api *API) RegisterAPIEndpoints(m *metrics.API) {
 		DataProxy: api.DataProxy,
 		ac:        api.AccessControl,
 	}
+	regularAuthzService := accesscontrol.NewRuleService(api.AccessControl)
```

Comment:
We can rename this since it doesn't accept the pluggable ActionsProvider anymore.

```suggestion
	ruleAuthzService := accesscontrol.NewRuleService(api.AccessControl)
```